llvm/test/LTO/AArch64/TestInputs/bar.ll

```diff
@@ -0,0 +1,35 @@
+; This file contains the new semantic of the branch-target-enforcement, sign-return-address.
+; Used for test mixing a mixed link case and also verify the import too in llc.
+
+; RUN: llc %s -o - | FileCheck %s
```

Comment:
I suppose it might be worth to add `-mattr=+pauth -mattr=+bti` to have actual pac/bti instruction in llc output rather than hint ones. Alternatively, you can leave current RUN line and add a new one. Also applies to other test files.

Reply:
ack.